internal/store/store.go

```diff
@@ -330,6 +333,21 @@ func (s *Store) GetBlueprintChanges(name string) []blueprint.Change {
 	return changes
 }
 
+func bumpVersion(str string) string {
+	v := [3]uint64{}
+	fields := strings.SplitN(str, ".", 3)
+	for i := 0; i < len(fields); i++ {
+		if n, err := strconv.ParseUint(fields[i], 10, 64); err == nil {
+			v[i] = n
+		} else {
+			// don't touch strings with invalid versions
+			return str
```

Comment:
In lorax-composer it runs the versions through semver.Version() in order to check their validity, raising an error if they are not compatible. I'm worried that by passing through invalid versions that they may end up in the saved blueprint and cause problems in the future.

Reply:
It does indeed! Thanks for the pointer. I think we should do the same here (filed as https://github.com/osbuild/osbuild-composer/issues/141). Since this PR doesn't regress on that, but rather moves a step in the right direction (being more compatible with lorax-composer by bumping the version), and it unblocks Red Hat QE, I'll merge it anyway.